testing/test_monkeypatch.py

```diff
@@ -192,6 +194,40 @@ def test_delenv():
             del os.environ[name]
 
 
+@pytest.mark.skipif(six.PY3, reason="python 2 only bug")
+class TestPython2BytesEnvironKeys(object):
+    """
+    In Python 2, os.environ keys need to be bytes, otherwise this will cause problems with other modules (notably
```

Comment:
this is specifically a problem for windows, should probably call that out?

Reply:
Will do